lib/components/SelectionView.js

```diff
@@ -54,7 +54,7 @@ class ViewButton extends PureComponent {
 class SelectionView extends PureComponent {
   static get propTypes() {
     return {
-      children: PropTypes.function,
+      children: PropTypes.array,
```

Comment:
This isn't a huge deal, but I find it nice when possible to add more specificity. This would be something like `PropTypes.arrayOf(PropTypes.node)`.

Reply:
not a deal breaker! Just a nice way to have the code be self documenting (and also better error messaging in case something went wrong)